spec/models/student_loans/eligibility_spec.rb

```diff
@@ -116,29 +129,64 @@
     end
   end
 
+  context "when saving in the “subjects-taught” context" do
+    it "is not valid if none of the subjects-taught attributes are true" do
+      expect(StudentLoans::Eligibility.new).not_to be_valid(:"subjects-taught")
+      expect(StudentLoans::Eligibility.new(biology_taught: false)).not_to be_valid(:"subjects-taught")
+      expect(StudentLoans::Eligibility.new(biology_taught: false, physics_taught: false)).not_to be_valid(:"subjects-taught")
+    end
+
+    it "validates the when one or more of the subjects-taught attributes are true" do
```

Comment:
Typo? I'm not sure what this is supposed to say.

Reply:
👍🏽